spec.html

```diff
@@ -7776,6 +7776,101 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
       <p>Host environments are not required to treat Jobs uniformly with respect to scheduling. For example, web browsers and Node.js treat Promise-handling Jobs as a higher priority than other work; future features may add Jobs that are not treated at such a high priority.</p>
     </emu-note>
 
+    <emu-clause id="sec-jobcallback-records">
+      <h1>JobCallback Records</h1>
+      <p>The JobCallback is a Record value used to store a callable object and a host-defined value. Callable objects that are invoked via a Job enqueued by the host may have additional host-defined context. To propagate the state, Job Abstract Closures should not capture and call callable objects directly. Instead, use HostMakeJobCallback and HostCallJobCallback (or its convenience wrapper CallJobCallback).</p>
```

Comment:
```suggestion
      <p>A <dfn>JobCallback Record</dfn> is a Record value used to store a callable object and a host-defined value. Callable objects that are invoked via a Job enqueued by the host may have additional host-defined context. To propagate the state, Job Abstract Closures should not capture and call callable objects directly. Instead, use HostMakeJobCallback and HostCallJobCallback (or its convenience wrapper CallJobCallback).</p>
```

Reply:
Also, "callable object" is a little ambiguous. Are abstract closures callable objects? Hopefully not. This should perhaps be "function object" everywhere in this PR. ("[A function object is an object that supports the [[Call]] internal method.](https://tc39.es/ecma262/#function-object)")

Reply:
Happy to change to callable object, though we also have wording like "a function is a callable object" in [Overview](https://tc39.es/ecma262/#sec-ecmascript-overview).